source/renderer/app/api/api.ts

```diff
@@ -2931,6 +2934,39 @@ export default class AdaApi {
     fakeStakePoolsJson: Array<StakePool>
   ) => void;
   setStakePoolsFetchingFailed: () => void;
+  getCatalystFund = async (): Promise<CatalystFund> => {
+    logger.debug('AdaApi::getCatalystFund called', {});
+
+    try {
+      const catalystFundRaw = await getCatalystFund();
+      const catalystFund: GetCatalystFundResponse = JSON.parse(catalystFundRaw);
+
+      logger.debug('AdaApi::getCatalystFund success', {
+        catalystFund,
+      });
+      return {
```

Comment:
maybe we could reshape a bit the returned to model to something like this:

```js
{
  current: {
    name: number,  
    startTime: Date,
    endTime: Date,
    results: Date
  },
  next: {
    name: number,  
    startTime: Date,
    registrationSnapshotTime: Date
  },
  registrationSnapshotTime: Date
}
```

This way you could remove the prefix `fund` and `next` from the properties and cluster them together.

Reply:
Sounds good, I'll update!